### PR TITLE
[ADD] Added new module account_invoice_report_stock_lot

### PR DESCRIPTION
**Pull request by krunal267**
Task URL:

https://www.druidoo.io/web#id=1821&action=367&model=project.task&view_type=form&menu_id=237

Module developed: account_invoice_report_stock_lot

### CODE REVIEW

File: account_invoice_report_stock_lot/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+====================================
+Stock Lots details in Invoice Report
+====================================
+
+This module adds details of Stock Lots in Invoice report.
```
Comment (ivantodorovich) [inline account_invoice_report_stock_lot/readme/DESCRIPTION.rst:5]:
```suggestion
This module adds details of Stock Lots in Invoice report.

It backports a feature that'a available in Odoo 13.0
```

File: account_invoice_report_stock_lot/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'summary': '',
+    'version': '12.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'author': "Druidoo",
+    'website': 'http://www.druidoo.io',
+    'license': "AGPL-3",
+    'depends': [
+        'account',
+        'sale',
+        'stock',
```
Comment (ivantodorovich) [inline account_invoice_report_stock_lot/__manifest__.py]:
```suggestion
        'sale_stock',
```

File: account_invoice_report_stock_lot/models/res_config_settings.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    group_lot_on_invoice = fields.Boolean(
+        "Display Lots & Serial Numbers on Invoices",
+        implied_group='account_invoice_report_stock_lot.group_lot_on_invoice'
+    )
```
Comment (ivantodorovich) [inline account_invoice_report_stock_lot/models/res_config_settings.py]:
```suggestion
    group_lot_on_invoice = fields.Boolean(
        "Display Lots & Serial Numbers on Invoices",
        help="Lots & Serial numbers will appear on the invoice",
        implied_group="account_invoice_report_stock_lot.group_lot_on_invoice",
    )
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Minor comments, otherwise looks great :)

Can you please generate a README.rst file?
Unfortunately we don't have the oca-github-bot running on our repositories (not yet..), so README.rst files are not auto generated :(

File: account_invoice_report_stock_lot/models/res_config_settings.py (hunk 3 above)
Comment (krunal267) [reply account_invoice_report_stock_lot/models/res_config_settings.py]: Okay @ivantodorovich 

noted it.

Comment (krunal267) [reply account_invoice_report_stock_lot/models/res_config_settings.py]: Hi @ivantodorovich 

I just pushed your suggested changes.

**Pull request merged**